app/src/main/res/drawable/ic_notoemoji.xml

```diff
@@ -0,0 +1,21 @@
+<vector android:height="24dp" android:viewportHeight="128"
+    android:viewportWidth="128" android:width="24dp"
```

Comment:
give it width/height of 34dp or Android 5 renders it blurry

Reply:
Is it enough if I only do that for the non-v24 version?

Reply:
yes I think so, but you can do it for both to be sure